ml-agents/mlagents/trainers/tests/torch/test_networks.py

```diff
@@ -87,8 +87,7 @@ def test_networkbody_visual():
         loss.backward()
         optimizer.step()
     # In the last step, values should be close to 1
-    for _enc in encoded.flatten():
-        assert _enc == pytest.approx(1.0, abs=0.1)
+    assert torch.mean(encoded) == pytest.approx(1.0, abs=0.1)
```

Comment:
I am not sure this is what we want, having the mean close to 1 does not mean the values are all close to 1.
Maybe 
```suggestion
    assert torch.sum((encoded - 1)**2) == pytest.approx(0.0, abs=0.01)
```

Reply:
I reverted it to the original, which does test every element.